pkg/controller/storagecluster/uninstall_reconciler_test.go

```diff
@@ -49,3 +52,57 @@ func assertStorageClusterUninstallAnnotation(
 		assert.Equal(t, string(CleanupPolicy), val)
 	}
 }
+
+func TestSetRookUninstallandCleanupPolicy(t *testing.T) {
+
+	for _, eachPlatform := range allPlatforms {
+		cp := &CloudPlatform{platform: eachPlatform}
+		t, reconciler, sc, _ := initStorageClusterResourceCreateUpdateTestWithPlatform(t, cp, nil)
+
+		// there are two annotations which will be 4 combinations, test all 4 combinations
```

Comment:
Can't we parametrize the 4 cases? creating tables (arrays) with the values of the annotations and the inputs for the corresponding assert function call? and then loop over the tables instead of writing all 4 cases out explicitly?

Reply:
Done thanks for the review.